tests/scripts/run-test-suites.pl

```diff
@@ -1,24 +1,76 @@
 #!/usr/bin/env perl
 
+# run-test-suites.pl
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2015-2018, ARM Limited, All Rights Reserved
```

Comment:
It's sort of funny the 2.1 backport has a newer copyright year than the other branches.

Reply:
That's because I wasn't paying attention on the other branches, but here there was no documentation at all so I adapted it. I'll amend the other PRs.